src/controllers/DevicesController.js

```diff
@@ -159,14 +164,20 @@ class DevicesController extends Controller {
       return this.ok({ id: deviceID, status: flashStatus });
     }
 
+    // 4
     // If signal exists then we want to toggle nyan mode. This just makes the
     // LED change colors.
-    if (!Number.isNaN(postBody.signal)) {
+    if (postBody.signal) {
+      if (postBody.signal !== ('1' || '0')) {
```

Comment:
This won't actually work the way you expect.

This would evaluate to:
`postBody.signal !== '1'` every time.

What you want is:
`!['1',  '0'].includes(postBody.signal)`